pkg/flypg/zombie.go

```diff
@@ -0,0 +1,84 @@
+package flypg
+
+import (
+	"errors"
+	"io/ioutil"
+	"os"
+)
+
+func ZombieLockExists() bool {
+	_, err := os.Stat("/data/zombie.lock")
+	if os.IsNotExist(err) {
+		return false
+	}
+	return true
+}
+
+func writeZombieLock(hostname string) error {
+	if err := ioutil.WriteFile("/data/zombie.lock", []byte(hostname), 0644); err != nil {
+		return err
+	}
+
+	return nil
+}
```

Comment:
Minor nit. `ioutil` is deprecated. You can use `io.WriteFile()` instead.

Reply:
Oh, right.  Are you talking about `os.WriteFile` ?